webpack.config.js

```diff
@@ -103,26 +103,27 @@ var info = {
 
         "systemd/services": [
             "systemd/init.js",
-            "systemd/services.css",
+            "systemd/services.less",
```

Comment:
Admittedly I never truly understood this, but my idea was that *either* you use `<link>` in the HTML and have to explicitly mention it here, *or* you `import` a less/css file in *.js, and then webpack already knows about it and it shouldn't be here. So could these perhaps be cleaned up now?

Reply:
This tell to bundle that file and then we link the imported bundled file in the CSS. Admitedly this is not the cleanest way to do this.
I suggest I port everything to use style-loader (which we already have installed in our deps but we don't use it_
In this way we can stop linking the bundled css in the html files, since the style loader does exactly that :)

Reply:
If that's easy to do, or we can do it separately even, then sure. I didn't say that it's wrong or that it needs to happen in this PR (in fact, if possible, it should not); I just don't understand it enough, and wanted to make sure that this is still justified. :-)